pyproject.toml

```diff
@@ -11,7 +11,7 @@ max-line-length = 120
 disable = "C0114, R1705, C0103"
 
 [tool.pytest.ini_options]
-addopts = "-vv --cov=. --cov-report term-missing --cov-fail-under 98 --benchmark-compare=0001 --benchmark-compare-fail=mean:30%"
+addopts = "-vv --cov=. --cov-report term-missing --cov-fail-under 98 --benchmark-storage=.benchmarks/Linux-CPython-3.9-64bit/ --benchmark-compare=0001 --benchmark-compare-fail=mean:30% --benchmark-columns='mean,stddev,median,iqr,outliers'"
```

Comment:
We need to specify the `--benchmark-storage` b/c this is being run in multiple OS'es in Github actions. So, for example, we want a Python 3.7 to look at the same benchmarks as Python 3.9 (they are close enough) rather than looking in it's default dir (which would be something like `Linux-CPython-3.7-64bit`).